apps/dolly-frontend/src/main/js/src/api/index.ts

```diff
@@ -32,11 +32,11 @@ export const multiFetcherAll = (urlListe, headers = null) => {
 	)
 }
 
-export const multiFetcherFagsystemer = (miljoUrlListe, headers = null) => {
+export const multiFetcherFagsystemer = (miljoUrlListe, headers = null, path = null) => {
 	return Promise.all(
 		miljoUrlListe.map((obj) =>
 			fetcher(obj.url, headers).then((result) => {
-				return { miljo: obj.miljo, data: [...result] }
+				return { miljo: obj.miljo, data: path ? result[path] : result }
```

Comment:
Fungerer multifetcheren fortsatt for de andre nå som data ser ut til å være objekt istedenfor array? 😅

Reply:
Om du husker at jeg klagde over noe som fungerte selv om det ikke burde fungere... det var dette 🤪 Men skal ta en dobbeltsjekk 😁

Reply:
Åja haha, husker den [...result] var litt hacky så knall hvis det funker uten 😄

Reply:
Fungerer faktisk helt amazing! 😎